pkg/ccl/backupccl/restore_job.go

```diff
@@ -221,6 +223,65 @@ func makeBackupLocalityMap(
 	return backupLocalityMap, nil
 }
 
+func filterCompletedImportSpans(
```

Comment:
nit: could you add a docstring which explains what this function does?

Reply:
done!